interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day/ajax_template.html

```diff
@@ -411,10 +412,15 @@
 echo "<td id='times'><div><table>\n";
 echo "<td class='timeslot providerXbtn' data-username='__PC_ALL__'><i class='fa fa-lg fa-user-md'></i></td>";
 //============================================================================================================================
+// Check global time preference so that 24 and 12hour preference is displayed properly
+$timeformat =0;
+if( $globle_timeformat == 1){ $timeformat = 12; }
```

Comment:
Wouldn't it be simpler to just use $GLOBALS['time_display_format'] here and drop the line of code at the top?